docs/src/components/GestureExamples/TapExample/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import { StyleSheet } from 'react-native';
+import {
+  GestureHandlerRootView,
+  GestureDetector,
+  Gesture,
+} from 'react-native-gesture-handler';
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+import Hand from '@site/static/img/hand-two.svg';
+import stylesWeb from './styles.module.css';
+import { RADIUS, useStylesForExample } from '../utils';
+
+export default function TapExample() {
+  const colorModeStyles = useStylesForExample();
+  const pressed = useSharedValue(false);
+  const [showHand, setShowHand] = useState(true);
+
+  const tap = Gesture.Tap()
+    .onBegin(() => {
+      setShowHand(false);
+      pressed.value = true;
+    })
+    .onEnd(() => {
+      setShowHand(true);
+    })
+    .onFinalize(() => {
+      pressed.value = false;
+    });
+
+  const animatedStyles = useAnimatedStyle(() => ({
+    backgroundColor: pressed.value
+      ? 'var(--swm-yellow-dark-80)'
+      : 'var(--swm-purple-light-100)',
+    transform: [{ scale: withTiming(pressed.value ? 1.2 : 1) }],
```

Comment:
```suggestion
    transform: [{ scale: withTiming(pressed.value ? 1.8 : 1) }],
```
lets make it a bit bigger coz I have big fingers that fully cover the button :)